packages/nextjs/hooks/scaffold-eth/useAllowedNetwork.ts

```diff
@@ -0,0 +1,8 @@
+import scaffoldConfig from "~~/scaffold.config";
+import { useGlobalState } from "~~/services/store/store";
+import { AllowedChainIds } from "~~/utils/scaffold-eth";
+
+export function useAllowedNetwork(chainId?: AllowedChainIds) {
```

Comment:
I think `useSelectedNetwork` is better for this hook since it returns selected network